src/psyclone/gocean1p0.py

```diff
@@ -644,23 +643,27 @@ def add_bounds(bound_info):
         gocean1.0 API is for a certain offset type and field type. It defines
         the loop boundaries for the outer and inner loop. The format is a
         ":" separated tuple:
-        bound_info = offset-type:field-type:iteration-space:outer-start:
-                      outer-stop:inner-start:inner-stop
+
+        > bound_info = offset-type:field-type:iteration-space:outer-start:
+        >             outer-stop:inner-start:inner-stop``
```

Comment:
Spurious "``" at end of line and line breaks in the example code aren't rendering correctly.

Reply:
The ">>>" syntax is the closer I could get to render them as expected. Would you do it differently?

Reply:
No, that looks fine to me.